### PR TITLE
Feat: zero beta portfolio

### PR DESCRIPTION
**Pull request by ShynRou**
Resolves https://sovryn.monday.com/boards/2218344956/pulses/2507858908

NUE no longer needed to be added.

### CODE REVIEW

File: src/app/components/SovGenerationNFTS/index.tsx [hunk 1]
```diff
@@ -87,49 +92,62 @@ export function SovGenerationNFTS() {
...
+                    <td>
+                      <div>{t(translations.sovGenerationNFTS.tierSovOG)}</div>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+        ))}
+        {!items?.length && !tiers?.length && balanceBday === '0' && (
```
Comment (creed-victor) [inline src/app/components/SovGenerationNFTS/index.tsx:139]: Maybe also add `loading` check from `useLoadSovNfts` and `useCacheCallWithValue` (for community tiers). Currently it shows that user has no NFTs while it's being loaded

File: src/utils/blockchain/contracts.testnet.ts [hunk 2]
```diff
@@ -333,4 +334,12 @@ export const contracts = {
     abi: marginTokenAbi,
     chainId: ChainId.BSC_TESTNET,
   },
+  ZUSD_token: {
+    address: '0x4a0741FA749eD6b1F810224D09f1f511952e67de',
+    abi: ZUSDAbi,
```
Comment (creed-victor) [inline src/utils/blockchain/contracts.testnet.ts:339]: At this point dapp only supports ERC20 capabilities for ZUSD token, so you can just reuse erc20TokenAbi

File: src/utils/blockchain/contracts.ts [hunk 3]
```diff
@@ -44,6 +44,7 @@ import fastBtcMultisigAbi from './abi/fastBtcMultisig.json';
 import perpetualManagerAbi from './abi/PerpetualManager.json';
 import perpetualLimitOrderBookAbi from './abi/PerpetualLimitOrderBook.json';
 import marginTokenAbi from './abi/MarginToken.json';
+import ZUSDAbi from './abi/ZUSDToken.json';
```
Comment (creed-victor) [inline src/utils/blockchain/contracts.ts:47]:
```suggestion
```

File: src/utils/blockchain/contracts.ts [hunk 4]
```diff
@@ -345,4 +346,14 @@ export const contracts = {
     abi: marginTokenAbi,
     chainId: ChainId.BSC_MAINNET,
   },
+
+  // TODO: add ZUSD and NUE mainnet contract addresses
+  ZUSD_token: {
+    address: '0x4a0741FA749eD6b1F810224D09f1f511952e67de',
+    abi: ZUSDAbi,
```
Comment (creed-victor) [inline src/utils/blockchain/contracts.ts:353]:
```suggestion
    abi: erc20TokenAbi,
```

File: src/utils/blockchain/contracts.testnet.ts [hunk 5]
```diff
@@ -41,6 +41,7 @@ import perpetualLimitOrderBookAbi from './abi/PerpetualLimitOrderBook.json';
 import marginTokenAbi from './abi/MarginToken.json';
 import fastBtcBridgeAbi from './abi/fastBtcBridge.json';
 import fastBtcMultisigAbi from './abi/fastBtcMultisig.json';
+import ZUSDAbi from './abi/ZUSDToken.json';
```
Comment (creed-victor) [inline src/utils/blockchain/contracts.testnet.ts:44]:
```suggestion
```

File: src/utils/blockchain/contracts.testnet.ts (hunk 2 above)
Comment (creed-victor) [inline src/utils/blockchain/contracts.testnet.ts:339]:
```suggestion
    abi: erc20TokenAbi,
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: src/app/components/UserAssets/BridgeLink.tsx [hunk 6]
```diff
@@ -104,6 +111,10 @@ export const BridgeLink: React.FC<IBridgeLinkProps> = ({ asset }) => {
             pathname: '/cross-chain/withdraw',
             state: { receiver, asset },
           }}
+          className={classNames(
+            styles.actionLink,
+            disableWithdrawal && styles.disabled,
+          )}
```
Comment (tiltom) [inline src/app/components/UserAssets/BridgeLink.tsx:117]:
```suggestion
          className={classNames(styles.actionLink, {
            [styles.disabled]: disableWithdrawal,
          })}
```

File: src/app/components/UserAssets/index.tsx [hunk 7]
```diff
@@ -321,7 +320,13 @@ const AssetRow: React.FC<IAssetRowProps> = ({
                   </div>
                 </Tooltip>
               ) : (
-                <Link to="/fast-btc/withdraw">
+                <Link
+                  to="/fast-btc/withdraw"
+                  className={classNames(
+                    styles.actionLink,
+                    tokens === '0' && styles.disabled,
+                  )}
```
Comment (tiltom) [inline src/app/components/UserAssets/index.tsx:328]:
```suggestion
                  className={classNames(styles.actionLink, {
                    [styles.disabled]: tokens === '0',
                  })}
```

File: src/app/components/UserAssets/index.tsx [hunk 8]
```diff
@@ -331,15 +336,22 @@ const AssetRow: React.FC<IAssetRowProps> = ({
             <button
               className={styles.actionLink}
               onClick={() => onConvert(item.asset)}
+              disabled={tokens === '0'}
```
Comment (tiltom) [inline src/app/components/UserAssets/index.tsx:339]: You use this condition (`tokens === '0'`) on many places, extract it to a variable (e.g. `hasAnyTokens`) and use `useMemo`.

**Review by tiltom — CHANGES_REQUESTED** [review]

**Review by tiltom — APPROVED** [review]
The code itself looks good but we shouldn't merge it until https://github.com/DistributedCollective/Sovryn-frontend/pull/2224 is in `development` (and its story - https://sovryn.monday.com/boards/2218344956/views/53321005/pulses/2629468226 closed) because it contains many adjustments for Portfolio page.

**Pull request closed without merging**